### PR TITLE
docs: component token meta

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

补一下组件 token 的元信息和文档

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 9895e7a</samp>

This pull request enhances the documentation and the functionality of the component token system. It adds JSDoc comments for various tokens in the `ComponentToken` interfaces of the menu, anchor, avatar, and breadcrumb components. It also improves the code style and the appearance of the color chunk component, and adds support for displaying and sorting global and component tokens in the component token table component. Additionally, it updates the `generate-token-meta` script to handle component-level tokens and to exclude deprecated tokens.

### <samp>🤖 Generated by Copilot at 9895e7a</samp>

*  Add JSDoc comments for component tokens in Chinese and English ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-317d329fe73e0dd37dc2416f47e9ddb7f377ece859e71f5f36a65025e7f8581eL7-R15), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-9bae49959a354c48caf848def36049b3c0d1e929e594370e57fc1a40c8a10f38L7-R50), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-4fbc88ad11a7132e8158c1091bf3a4f6c0c2c3eab7b56e38cd6fe198fede233dL7-R40), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL14-R22), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dR28-R31), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL25-R48), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL35-R90), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dR96-R99), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL61-R140), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL82-R164), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL95-R189), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dL108-R230))
*  Modify `generate-token-meta` script to generate token meta data for global and component tokens ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-34a461a77ece7f7d9b58b82cf46afb2c956ca4dc970b51bc4cfe936280f9badbL5-R18), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-34a461a77ece7f7d9b58b82cf46afb2c956ca4dc970b51bc4cfe936280f9badbL43-R53), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-34a461a77ece7f7d9b58b82cf46afb2c956ca4dc970b51bc4cfe936280f9badbL51-R128))
*  Add `ComponentTokenTable` component to render component tokens and global tokens in separate sub tables ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93R20-R21), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93R28-R29), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L51-R58), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L67-R93), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L92-R103), [link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-377ed68f4e882f6438c31331026bdb11548e6b62be65727a5793cda37badfa93L125-R160))
*  Modify `TokenTable` component to use `tokenMeta.global` object for filtering and mapping global tokens ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-16fa19cd7384aaec1f2aa2733ad3a35affb9541febf0bd5aaa779eb0c7d9890aL101-R101))
*  Modify `ColorChunk` component to use fixed percentage for `border-radius` ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L26-R26))
*  Change import order of React and other modules to follow ESLint rule ([link](https://github.com/ant-design/ant-design/pull/43337/files?diff=unified&w=0#diff-51d36d0662438e162135ad6713e967a78491b32e953192265b2ecc71c9800e26L1-R3))

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: components/typography/style/index.ts [hunk 1]
```diff
@@ -126,9 +133,8 @@ const genTypographyStyle: GenerateStyle<TypographyToken> = (token) => {
 export default genComponentStyleHook(
   'Typography',
   (token) => [genTypographyStyle(token)],
-  (token) => ({
+  () => ({
     titleMarginTop: '1.2em',
     titleMarginBottom: '0.5em',
-    fontWeightStrong: token.fontWeightStrong,
```
Comment (li-jia-nan) [inline components/typography/style/index.ts:132]: 这个不要了吗？

Comment (MadCcc) [reply components/typography/style/index.ts:132]: 这个没用

File: components/input-number/style/index.ts [hunk 2]
```diff
@@ -15,10 +15,26 @@ import type { FullToken, GenerateStyle } from '../../theme/internal';
...
+  /**
+   * @desc 操作按钮图标大小
+   * @descEN Icon size of control button
+   */
   handleFontSize: number;
-  /** Default `auto`. Set `true` will always show the handle */
+  /**
+   * Default `auto`. Set `true` will always show the handle
+   * @desc 操作按钮可见性
+   * @descEN Handle visible
```
Comment (afc163) [inline components/input-number/style/index.ts:36]: Handler visibility

**Comment by afc163** [comment]
还是需要补一个通用使用说明，我记得之间提过。

**Pull request merged**